AppLovin MAX Demo App - Swift/AppLovin MAX Demo App - Swift/Base Classes/MANativeTemplateAdViewSwiftUIWrapper.swift

```diff
@@ -0,0 +1,143 @@
+//
+//  MANativeTemplateAdViewSwiftUIWrapper.swift
+//  AppLovin MAX Demo App - Swift
+//
+//  Created by Matthew Nguyen on 7/31/23.
+//  Copyright © 2023 AppLovin. All rights reserved.
+//
+
+import SwiftUI
+import AppLovinSDK
+
+@available(iOS 13.0, *)
+struct MANativeTemplateAdViewSwiftUIWrapper: UIViewRepresentable
+{
+    @Binding var triggerShowAd: Bool
+    let nativeAdLoader: NativeSwiftUIAdLoader
+    let containerView: UIView
+    
+    // MANativeAdDelegate methods
+    var didLoadNativeAd: ((MANativeAdView?, MAAd) -> Void)? = nil
+    var didFailToLoadNativeAd: ((String, MAError) -> Void)? = nil
+    var didClickNativeAd: ((MAAd) -> Void)? = nil
+    var didExpireNativeAd: ((MAAd) -> Void)? = nil
+    
+    // MAAdRevenueDelegate methods
+    var didPayRevenue: ((MAAd) -> Void)? = nil
+        
+    
+    func makeUIView(context: Context) -> UIView
+    {
+        nativeAdLoader.setNativeAdDelegate(context)
+        nativeAdLoader.setRevenueDelegate(context)
+        
+        return containerView
+    }
+    
+    func updateUIView(_ uiView: UIView, context: Context)
+    {
+        if triggerShowAd
+        {
+            nativeAdLoader.showAd()
+        }
+    }
+    
+    func makeCoordinator() -> Coordinator
+    {
+        Coordinator(parent: self)
+    }
+}
+
+@available(iOS 13.0, *)
+extension MANativeTemplateAdViewSwiftUIWrapper
+{
+    class Coordinator: NSObject, MANativeAdDelegate, MAAdRevenueDelegate
+    {
+        private let parent: MANativeTemplateAdViewSwiftUIWrapper
+        
+        init(parent: MANativeTemplateAdViewSwiftUIWrapper)
+        {
+            self.parent = parent
+        }
+        
+        // MANativeAdDelegate methods
+        func didLoadNativeAd(_ maxNativeAdView: MANativeAdView?, for ad: MAAd)
+        {
+            parent.didLoadNativeAd?(maxNativeAdView, ad)
+        }
+        
+        func didFailToLoadNativeAd(forAdUnitIdentifier adUnitIdentifier: String, withError error: MAError)
+        {
+            parent.didFailToLoadNativeAd?(adUnitIdentifier, error)
+        }
+        
+        func didClickNativeAd(_ ad: MAAd)
+        {
+            parent.didClickNativeAd?(ad)
+        }
+        
+        func didExpireNativeAd(_ ad: MAAd)
+        {
+            parent.didExpireNativeAd?(ad)
+        }
+        
+        // MAAdRevenueDelegate methods
+        func didPayRevenue(for ad: MAAd)
+        {
+            parent.didPayRevenue?(ad)
+        }
+    }
+}
+
+@available(iOS 13.0, *)
+class NativeSwiftUIAdLoader
+{
+    var adUnitId: String
```

Comment:
Our convention on iOS is to write out `Identifier` and most other words, cause Apple's APIs tend to write them out
```suggestion
    var adUnitIdentifier: String
```